hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-config

```diff
@@ -47,6 +47,7 @@ OZONE-SITE.XML_ozone.scm.client.address=scm
 OZONE-SITE.XML_hdds.block.token.enabled=true
 OZONE-SITE.XML_hdds.grpc.tls.enabled=true
 OZONE-SITE.XML_ozone.replication=3
+OZONE-SITE.XML_hdds.scmclient.max.retry.timeout=30s
```

Comment:
This will be a problem for the secure tests, as getScmInfo will retry for 30s and fail. We want to try little longer for that as we see some failures in test

Reply:
Addressed.